aqua/ipfs-api.aqua

```diff
@@ -49,14 +49,17 @@ func get_and_cache(
 -- path should exist & be available to `aqua-ipfs`
 func put(node: PeerId, path: string) -> IpfsPutResult:
     on node:
-        result <- Ipfs.put(path)
-    <- result
+        <- Ipfs.put(path)
 
 -- Download file `cid` from IPFS node `from` and save it to `node`
 func get_from(node: PeerId, cid: CID, from: Multiaddr) -> IpfsGetResult:
     on node:
-        result <- Ipfs.get_from(cid, from)
-    <- result
+        <- Ipfs.get_from(cid, from)
+
+-- Download file `cid` from IPFS node `from` and save it to `node`
```

Comment:
Doc comment is the same as for get_from, it's a little confusing

Reply:
You should definitely mention that it also returns file contents

Reply:
leftover from copy-pasting. thx!